staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/types.go

```diff
@@ -30,22 +30,28 @@ import (
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 )
 
+const (
+	noMaxLength = -1
```

Comment:
Is it worth using nil to represent the absence of a maxLength and just using *int64s everywhere?  I'm worried a maxLength field is going to get zero valued somewhere on accident and we're going to have a `0` where we wanted a `-1` and it's going to mess things up.  If we instead use nil, then the zero value does the right thing.

Reply:
Since it means "infinity" maybe it's better to make it `LONG_LONG_MAX`?

Reply:
I like the idea of using the largest possible integer - I've changed the PR to use `math.MaxInt`, though I don't think it should be resolved just yet since I think using pointers may have merit too.

Reply:
I'm okay with either. Just remove any usages of -1 (there are a couple in the test code right now).